packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -18,7 +18,7 @@ export interface IDropdownProps extends ISelectableDroppableTextProps<HTMLDivEle
   /**
    * Callback issues when the selected option changes
    */
-  onChanged?: (option: IDropdownOption, index?: number) => void;

```

Comment:
We should not change this, this is a breaking change! Please, please do not break things. Add things, yes. Deprecate? sure. Don't remove props. If I am a customer which has a callback for onChanged which takes in an index 2nd parameter, I believe we will see a compiler break here.

Also specific to this area, we should be deprecating all instances of `onChanged` so that we consistently have an `onChange` event. It is currently inconsistent and needs to be scrubbed. That may be a better opportunity to refine the params that are passed to the callback.